podium/talks/models.py

```diff
@@ -20,11 +20,20 @@ class Talk(models.Model):
         max_length=1, choices=TALK_STATUS_CHOICES,
         default='S')
 
+    def get_absolute_url(self):
+        return f'/talks/talks/{self.id}/'
```

Comment:
This should use Django's built-in "reverse" to be more future-proof against possible URL pattern changes.
See the following:  https://docs.djangoproject.com/en/1.11/ref/urlresolvers/#reverse